src/components/display/fewlines/Header/Header.tsx

```diff
@@ -7,25 +7,25 @@ export const Header: React.FC = () => {
   return (
     <Flex>
       <img width="90" src={fewlinesLogo} aria-label="Fewlines logo" />
-      <H5>Account</H5>
+      <p>Account</p>
     </Flex>
   );
 };
 
 const Flex = styled.div`
   display: flex;
   align-items: center;
-  margin: ${({ theme }) => theme.spaces.xs};
-`;
+  margin: ${({ theme }) => `${theme.spaces.xs} ${theme.spaces.xs}`};
 
-const H5 = styled.h5`
-  padding-left: ${({ theme }) => theme.spaces.xxs};
-  margin-left: ${({ theme }) => theme.spaces.xxs};
-  border-left: ${({ theme }) =>
-    `${theme.colors.blacks[0]} ${theme.borders.thin}`};
-  font-weight: ${({ theme }) => theme.fontWeights.normal};
-  font-size: ${({ theme }) => theme.fontSizes.s};
-  display: flex;
-  align-items: center;
-  height: ${({ theme }) => theme.spaces.s};
+  p {
```

Comment:
So what's our stance here, are classes out of the way because it seems like all that padding and margin and font properties should be some kind of default or reuseable in some fashion, don't you think?

Reply:
I think we should avoid classes if we are using `styled-component`.

You are right in the sense that we should set CSS properties more globally. This is something I want to fix in the next sprints, and will probably be a good task for newcomers 😉.